pkg/client/fake/fake_client_test.go

```diff
@@ -2,22 +2,25 @@ package fake
 
 import (
 	"context"
-	"reflect"
 	"testing"
 
 	"github.com/docker/docker/errdefs"
 	"github.com/stretchr/testify/require"
+	"gotest.tools/assert"
 
 	composeTypes "github.com/docker/stacks/pkg/compose/types"
 	"github.com/docker/stacks/pkg/types"
 )
 
 var stackCreate = types.StackCreate{
-	Metadata: types.Metadata{
-		Name: "teststack",
-	},
 	Orchestrator: types.OrchestratorSwarm,
 	Spec: types.StackSpec{
+		Metadata: types.Metadata{
```

Comment:
just to be a ruthless pedant, what are the chances that we could rename this to `Annotations` in mirror of swarm's types? or would that put it odds with the kube model?

Reply:
Yeah, in Kubernetes you have an `ObjectMeta` which includes fields `Labels` and `Annotations` which are both string maps, but the latter is not intended for internal use at all. I actually went back and forth with this a little and I think that since the term `Annotations` is used in different contexts across the two orchestrators, it's preferable to avoid the term in favor of `Meta` and `Labels`

Reply:
that sounds fine.